CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
+## v0.8.1 [2022-02-24]
+
 ## v0.8.0 [2022-02-18]
 
+_Bug fixes_
+
+- Fixed the `oci_kms_key` table to list keys from different compartments instead of fetching keys from only the vault's compartment ([#365](https://github.com/turbot/steampipe-plugin-oci/pull/365))
+- The following tables have been renamed owing to the maximum character limit(63 bytes) set by PostgreSQL for identifiers. We recommend updating any scripts or workflows that use these tables to use the renamed equivalent tables instead ([#364](https://github.com/turbot/steampipe-plugin-oci/pull/364))
```

Comment:
```suggestion
- The following tables have been renamed to shorten their names to avoid hitting a PostgreSQL identifier character limit. Any scripts or workflows that use should be updated to use the updated names. ([#364](https://github.com/turbot/steampipe-plugin-oci/pull/364))
```